src/bioregistry/data/bioregistry.json

```diff
@@ -106932,6 +106941,35 @@
       "prefix": "sopharm"
     }
   },
+  "sosa": {
+    "contributor": {
+      "github": "StroemPhi",
+      "name": "Philip Strömert",
+      "orcid": "0000-0002-1595-3213"
+    },
+    "description": "This ontology is based on the SSN Ontology by the W3C Semantic Sensor Networks Incubator Group (SSN-XG), together with considerations from the W3C/OGC Spatial Data on the Web Working Group.",
+    "example": "FeatureOfInterest",
+    "github_request_issue": 1274,
+    "homepage": "https://www.w3.org/TR/vocab-ssn/",
+    "license": "http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document",
+    "name": "Sensor, Observation, Sample, and Actuator (SOSA) Ontology",
+    "publications": [
+      {
+        "doi": "10.1016/j.websem.2018.06.003",
+        "title": "SOSA: A lightweight ontology for sensors, observations, samples, and actuators",
+        "year": 2019
+      }
+    ],
+    "rdf_uri_format": "http://www.w3.org/ns/sosa/$1",
+    "repository": "https://github.com/w3c/sdw",
+    "reviewer": {
+      "email": "cthoyt@gmail.com",
+      "github": "cthoyt",
+      "name": "Charles Tapley Hoyt",
+      "orcid": "0000-0003-4423-4370"
+    },
+    "uri_format": "https://www.w3.org/TR/vocab-ssn/#$1"
```

Comment:
@StroemPhi I noticed that the SOSA w3.org URIs resolve to SSN that is the same as SSN in the other record. is this like a DC vs. DCTERMS situation?